vitest.config.ts

```diff
@@ -11,7 +11,7 @@ export default defineConfig({
 				root: `./packages/${name}`,
 				setupFiles: [
 					"console-fail-test/setup",
-					"../core/lib/ts-patch/install-patch-hooks.js",
+					"@flint.fyi/core/lib/ts-patch/install-patch-hooks.js",
```

Comment:
```plaintext
...

 FAIL   ts  src/rules/voidOperator.test.ts [ packages/ts/src/rules/voidOperator.test.ts ]
 FAIL   ts  src/rules/withStatements.test.ts [ packages/ts/src/rules/withStatements.test.ts ]
Error: Cannot find module '/home/runner/work/flint/flint/packages/ts/@flint.fyi/core/lib/ts-patch/install-patch-hooks.js'
```

Is this, ah, supposed to work?

Reply:
😅

Oh yeah, forgot to mention the semantic merge conflict with #1138. Just needs to be added to exports.